cloud-functions/comparer-function/main.py

```diff
@@ -32,7 +32,9 @@ def handle(request):
     previous_frame_path = download_blob(bucket_name, 'previous_frame.jpg')
     if previous_frame_path is not None:
 
-        percentage = compare(latest_frame_path, previous_frame_path)
+        image_mask = os.environ.get('IMAGE_MASK',None)
```

Comment:
The image won't be uploaded so I can offer some suggestions as to how we get the image available to the function:

1. We upload it to a bucket in terraform similar to how I've done with the cloud function code
2. We move the image local to the `main.py` and in theory it should get zipped up with the codebase you'll be bale to reference it from `main.py` as `mask.png`.

Reply:
Have moved mask.png into the comparer folder, let see how that goes